framework/src/media/media_recorder.c

```diff
@@ -268,7 +271,7 @@ int record_and_write(char *buffer)
 	int size;
 	char inst = ST_PROTO_RECORD_DATA;
 
-	frames = pcm_readi(g_rc.pcmin, buffer, g_rc.buffer_size);
+	frames = pcm_readi(g_rc.pcmin, buffer, pcm_bytes_to_frames(g_rc.pcmin, g_rc.buffer_size));
```

Comment:
Because this line was changed at #1125 , this causes conflict. What is your plan? Who will change this line?

Reply:
i will merge commit